cmd/lit-af/walletcmds.go

```diff
@@ -80,8 +80,8 @@ func (lc *litAfClient) Send(textArgs []string) error {
 
 // Sweep moves utxos with many 1-in-1-out txs
 func (lc *litAfClient) Sweep(textArgs []string) error {
-	err := CheckHelpCommand(sweepCommand, textArgs, 2)
-	if err != nil {
+	err, stopEx := CheckHelpCommand(sweepCommand, textArgs, 2)
+	if err != nil || stopEx {
```

Comment:
This will return a nil error if stopEx is true.  Is this the functionality we want?  Seems strange to break execution with no error.

Reply:
Well, stopEx checks whether the user has requested the help command. Since the user hasn't input something wrong, we don't return an error. Returning one will bring us back to the present case that we have :)

Reply:
In the old situation we would also return nil if the `-h` parameter was passed:

(one instance in an older version of this code:)

```
	if len(textArgs) > 0 && textArgs[0] == "-h" {
		fmt.Fprintf(color.Output, historyCommand.Format)
		fmt.Fprintf(color.Output, historyCommand.Description)
		return nil
	}
```

Reply:
OK that makes sense.  Not actually an "error"